raft/log.go

```diff
@@ -24,10 +24,10 @@ import (
 
 const (
 	// DefaultHeartbeatInterval is the default time to wait between heartbeats.
-	DefaultHeartbeatInterval = 50 * time.Millisecond
+	DefaultHeartbeatInterval = 150 * time.Millisecond
```

Comment:
Yeah, a lower heartbeat is easier for testing at this point and it spews out heartbeat errors at a third of the rate which is nice. :) we can adjust later.